ci-operator/step-registry/ipi/conf/aws/sharednetwork/ipi-conf-aws-sharednetwork-ref.yaml

```diff
@@ -11,3 +11,8 @@ ref:
       memory: 100Mi
   documentation: |-
     The IPI AWS configure step generates the AWS-specific install-config.yaml contents based on the cluster profile and optional input files.
+  env:
+  - name: ZONES_COUNT
+    default: "2"
+    documentation: |-
+      The number of AZs to present in the cluster. Accepted values are 1, 2, and 3.
```

Comment:
nit: "...number of Availability Zones to present..." would save readers the mental effort of unpacking the "AZ" acronym.